xxx.txt

```diff
@@ -17,3 +17,10 @@ GUUUUUUUUUUUUUUUUUUUUUUUUUUUU
 djfaljfa lfjel
 OOOOOO
 mmmmmmmmmmmmmmmmm
+it is mean of you to ridicule him in public.
+this is new line1
+this is new line2
```

Comment:
between line2 and line3
Add single comment

Reply:
this is reply